float64[6] m_xyz__f_xyz
